lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -480,6 +481,15 @@ sub _compose_job_overview_search_args ($c) {
     # allow filtering by group ID or group name
     $search_args{groupids} = [map { $_->id } @groups] if @groups;
 
+    # allow excluding jobs by group ID (0 to exclude groupless jobs)
+    if ($v->is_valid('not_groupid')) {
+        my @not_group_ids = @{$v->every_param('not_groupid')};
```

Comment:
It makes sense to avoid a copy in cases like this:
```suggestion
        my $not_group_ids = $v->every_param('not_groupid');
```
Of course you'll need to change the following lines to use `$not_group_ids` instead of `@not_group_ids`.